test/lint.t

```diff
@@ -28,10 +29,12 @@ Tests the following:
   * b-incorrect-opam (HEAD -> new-branch-1)
   * a-1 (master)
   $ opam-repo-ci-local --repo="." --branch=new-branch-1 --lint-only --no-web-server
-  Error "3 errors:
+  Error "5 errors:
   Error in b.0.0.1:            warning 25: Missing field 'authors'
   Error in b.0.0.2:              error  3: File format error in 'unknown-field' at line 11, column 0: Invalid field unknown-field
-  Error in b.0.0.3: pin-depends present. This is not allowed in the opam-repository."
+  Error in b.0.0.3: pin-depends present. This is not allowed in the opam-repository.
+  Error in system-b.0.0.1: package with prefix 'system-' requires conflict class 'ocaml-system-setting'
+  Warning in system-b.0.0.1: package name has restricted prefix 'system-'"
```

Comment:
Can you also include a test for the other conflict class error? `"Error in %s: package with conflict class '%s' requires name prefix '%s'"`